pino.js

```diff
@@ -9,6 +9,7 @@ var noop = require('./noop')
 var pid = process.pid
 var hostname = os.hostname()
 var baseLog = flatstr('{"pid":' + pid + ',"hostname":"' + hostname + '",')
+var extend = require('util')._extend
```

Comment:
There are diferences that I cannot remember right now. Ill have a look
again just to be sure.

On 26 Jul 2016 5:58 p.m., "David Mark Clements" <notifications@github.com>
wrote:

> In pino.js <https://github.com/mcollina/pino/pull/75#discussion_r72291396>
> :
>
> > @@ -9,6 +9,7 @@ var noop = require('./noop')
> >  var pid = process.pid
> >  var hostname = os.hostname()
> >  var baseLog = flatstr('{"pid":' + pid + ',"hostname":"' + hostname + '",')
> > +var extend = require('util')._extend
>
> Wait isn't this the same as Object.assign ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mcollina/pino/pull/75/files/446f62200f0db98b62c9b9bc5cb05656da29d9dd#r72291396>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAHkOhFDSUXvSuyzFdE00HhlJaduYxjBks5qZjykgaJpZM4JUo7K>
> .
>